app/controllers/provider/admin/dashboard/widget_controller.rb

```diff
@@ -28,7 +28,7 @@ def timeline_data(current_data, previous_data)
         previous:   previous_data
       },
       value:             current_sum,
-      current:           incomplete_slice.values.sum,
+      current:           incomplete_slice.values.first,
```

Comment:
So, this is confusing, but I think it's correct.

Basically, here I was trying to fix the following deprecation warning:
```
DEPRECATION WARNING: Rails 7.0 has deprecated Enumerable.sum in favor of Ruby's native implementation available since 2.4. Sum of non-numeric elements requires an initial argument. (called from timeline_data at /home/dmayorov/Projects/3scale/porta/app/controllers/provider/admin/dashboard/widget_controller.rb:31)
```

Sample values in this block:
```
incomplete_slice == {"2024-12-17"=>{:value=>1, :formatted_value=>"1"}}
incomplete_slice.values == [{:value=>1, :formatted_value=>"1"}]
incomplete_slice.values.sum == {:value=>1, :formatted_value=>"1"}
```

I believe `incomplete_slice` will always be a hash with a single key-value element, because it's using a single date - `current_data_keys.pop`.

In fact, I believe that if it wasn't a single element, it would actually throw an error, because someting like `[{:value=>1, :formatted_value=>"1"}, {:value=>1, :formatted_value=>"1"}].sum` would fail.